Description: Fix octave path for Debian
 In Debian we want octave-cli; octave is the gui
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/interfaces/octave.py
+++ b/sage/src/sage/interfaces/octave.py
@@ -184,7 +184,7 @@
         Expect.__init__(self,
                         name = 'octave',
                         prompt = '>',
-                        command = "sage-native-execute octave --no-line-editing --silent",
+                        command = "sage-native-execute octave-cli --no-line-editing --silent",
                         server = server,
                         server_tmpdir = server_tmpdir,
                         script_subdirectory = script_subdirectory,
